data/moves.js

```diff
@@ -9626,7 +9626,9 @@ let BattleMovedex = {
 		volatileStatus: 'magiccoat',
 		effect: {
 			duration: 1,
-			onStart: function (target) {
+			onStart: function (target, source, sourceEffect) {
+				// @ts-ignore
+				target.volatiles['magiccoat'].pranksterBoosted = sourceEffect.pranksterBoosted || false;
```

Comment:
``target.volatiles['magiccoat']`` is ``this.effectData``

Reply:
What's the difference?

Reply:
It's the same... Using ``this.effectData`` yields more portable code though.

Reply:
Alright, that answers the question I meant to ask.